### ISSUE TITLE
Only use CAT pool in e2e tests

### ISSUE THREAD

**Issue by ninabarbakadze** [celestiaorg/celestia-core#1230]
## Description

We should get rid of all mempools other than CAT in e2e tests

**Comment by rootulp** [comment]
Why? The prioritized mempool is still used on mainnet because it's the default. IMO we should have test coverage for the prioritized mempool at least until we change the default to CAT and deprecate the other two.

**Comment by cmwaters** [comment]
Agree. We should have tests for all implementations that we support

**Comment by ninabarbakadze** [comment]
I didn't specify that we should only do that once we've migrated to cat so thanks for bringing that up. will make sure to add this to the description. @rootulp

**Comment by rootulp** [comment]
thanks! After we migrate to CAT pool, should we delete the v0 and v1 mempools entirely? I don't see a rationale for deleting  just the e2e tests if we're not also deleting the actual implementation.

**Comment by ninabarbakadze** [comment]
I'd defer to @evan-forbes on this but generally I'd agree

**Comment by evan-forbes** [comment]
after mainnet switched to cat for a while we can yeah. idk how long of a period of time we'd need to be super confident that we don't need a backup

**Comment by rootulp** [comment]
IMO this issue is too narrow in scope. We should deprecate and delete the non-CAT mempools entirely, not just in e2e tests:

1. https://github.com/celestiaorg/celestia-core/issues/1382
2. https://github.com/celestiaorg/celestia-core/issues/2802

Once we remove those mempools, the Go compiler will force us to update the e2e tests to use the CAT mempool.
